content/chainguard/migration/migrating-php.md

```diff
@@ -42,7 +43,19 @@ Wolfi offers several PHP extensions as optional packages you can install with `a
 - `php-sodium`
 - `php-phar`
 
-To check for available extensions, you can run a temporary container based on the `php:latest-dev` image:
+In addition to those, the Laravel image includes the following extensions:
+- `php-ctype`
+- `php-dom`
+- `php-fileinfo`
+- `php-simplexml`
+
+You can run an ephemeral container with the `php -m` command to see a list of enabled extensions:
```

Comment:
I probably wouldn't say "ephemeral" here as ephemeral containers are something specific in k8s land. I'm not sure you need an adjective, but maybe "temporary"?